portlets/opencps-portlet/docroot/html/portlets/dossiermgt/frontoffice/edit_dossier.jsp

```diff
@@ -67,14 +72,29 @@
 	String[][] categorySections = {dossierSections};
 	
 	boolean isEditDossier = ParamUtil.getBoolean(request, "isEditDossier");
+	
+	
+	
+	
+	ProcessOrder processOrder = null;
+	ProcessWorkflow workFlow = null;
+	try {
+		if(Validator.isNotNull(dossier)) {
+			processOrder = ProcessOrderLocalServiceUtil.getProcessOrder(dossier.getDossierId(), 0);
+			workFlow = ProcessWorkflowLocalServiceUtil.getByS_PreP_AN(processOrder.getServiceProcessId(), processOrder.getProcessStepId(), "Thông báo hủy hồ sơ");
```

Comment:
Hàm này thấy sử dụng rất nhiều nơi. Cả java và jsp và toàn tương thẳng tiếng việt vào -> Tại sao không khai báo trong Properties rồi gọi ra dùng chung ??